### PR TITLE
RRD: rename Flipper flags to be prefixed with `rrd_` - step 1

### PR DESCRIPTION
**Pull request by yoomlam**
## Description of change

Add new Flipper flags `rrd_hypertension_compensation` and `rrd_add_special_issue` to replace `disability_hypertension_compensation_fast_track` and `disability_hypertension_compensation_fast_track_add_rrd`, respectively. The code is updated to look for both old and new Flipper flags for now.

Once the new Flipper flags are enabled in prod, follow-up with another PR to remove the old Flipper flags.

## Original issue(s)
Use prefix `rrd` to associate Flipper flags with our RRD team, and remove "fast_track".

## Things to know about this PR

This PR is the first step. The next step is to enable the new Flipper flags in prod and create a PR to remove the old Flipper flags.

### CODE REVIEW

File: app/services/rapid_ready_for_decision/processor_selector.rb [hunk 1]
```diff
@@ -34,8 +34,9 @@ def rrd_enabled?
 
     # @return [Boolean] Is the specified disability RRD-enabled according to Flipper settings
     def rrd_enabled_disability?(disability)
-      # Todo later: Rename RRD-related Flipper keys to "rrd_#{disability}_compensation"
-      Flipper.enabled?("disability_#{disability.downcase}_compensation_fast_track".to_sym)
+      # Todo later: Remove old RRD-related Flipper keys "disability_#{disability.downcase}_compensation_fast_track"
+      Flipper.enabled?("rrd_#{disability.downcase}_compensation".to_sym) ||
+        Flipper.enabled?("disability_#{disability.downcase}_compensation_fast_track".to_sym)
```
Comment (yoomlam) [inline app/services/rapid_ready_for_decision/processor_selector.rb:39]: Check for both old and new flags for this PR.

File: config/features.yml [hunk 2]
```diff
@@ -407,6 +407,14 @@ features:
...
     description: Ask for a password when an encrypted PDF is detected before uploading
     enable_in_development: true
+  rrd_hypertension_compensation:
+    actor_type: user
+    description: Fast tracks 526 disability compensation hypertension claims by submitting additional health data
+    enable_in_development: true
+  rrd_add_special_issue:
+    actor_type: user
+    description: Fast tracks 526 disability compensation claims by adding the RRD special issue to the claim.
+    enable_in_development: true
```
Comment (yoomlam) [inline config/features.yml:417]: Copied from `disability_hypertension_compensation_fast_track` and `disability_hypertension_compensation_fast_track_add_rrd`, respectively.

File: config/features.yml [hunk 3]
```diff
@@ -191,10 +191,6 @@ features:
     actor_type: user
     description: Fast tracks 526 disability compensation hypertension claims by submitting additional health data
     enable_in_development: true
-  rrd_asthma_compensation:
-    actor_type: user
-    description: Fast tracks 526 disability compensation asthma claims by submitting additional health data
-    enable_in_development: true
```
Comment (yoomlam) [inline config/features.yml:197]: Moved down to alphabetize flipper names and keep RRD-related Flipper flags together

**Review by nanotone — APPROVED** [review]
Looks great!

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**